classification.yaml

```diff
@@ -0,0 +1,791 @@
+DMS-MWBT-REQ-0033:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Use a butler to read a subimage via get parameters against any repo.
+DMS-MWBT-REQ-0034:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Use a butler to read a component (e.g. WCS), against any repo.
+DMS-MWBT-REQ-0035:
+  status: unclear
+  priority: 1a
+  notes:
+    This is not a middleware responsibility; it's a feature of whatever in-memory dataset one is trying to operate on (e.g. afw classes).
+DMS-MWBT-REQ-0040:
+  status: feature-incomplete
+  priority: 1a
+  missing:
+    - Tabular-data Datastore, backed by opaque tables.
+  script:
+    - Create or find repo with tabular data in database.
+    - Create or find repo with S3 storage.
+    - Do Butler.get.
+DMS-MWBT-REQ-0045:
+  status: feature-incomplete
+  priority: 1a
+  missing:
+    - Some kind of rendezvous with EFD.
+DMS-MWBT-REQ-0046:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Run ci_hsc_gen3.
+  jira:
+  - LVV-T1982
+  - LVV-T1985
+DMS-MWBT-REQ-0047:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Run ci_hsc_gen3.
+  jira:
+  - LVV-T1982
+DMS-MWBT-REQ-0048:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Make an empty repo.
+    - Ingest some external parquet or FITS catalog.
+    - Call butler.put.
+DMS-MWBT-REQ-0052:
+  status: unclear
+  priority: 1b
+  notes:
+    We aren't planning to need this for batch purposes anymore.  It's still something we could add on top of existing functionality, but it's not so easy to do so that we would want to implement it just for verification.
+DMS-MWBT-REQ-0053:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Manually create QG with execution butler.
+    - Run butler.get against execution butler.
+  notes:
+    - Execution butler may be removed in favor of something better (prototyping on DM-32072) by the time we run this test, but whatever replaces it will also satisfy this requirement.
+DMS-MWBT-REQ-0054:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Manually create QG with execution butler.
+    - Run butler.get against execution butler for ref that does not exist.
+  notes:
+    - Execution butler may be removed in favor of something better (prototyping on DM-32072) by the time we run this test, but whatever replaces it will also satisfy this requirement.
+DMS-MWBT-REQ-0055:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Enable datastore caching in a Butler client in RSP (or any S3-backed repo).
+    - Run butler.get twice, check (e.g. trace logs) that the second comes from cache.
+DMS-MWBT-REQ-0056:
+  status: unclear
+  priority: 1a
+  notes:
+    Use cases like the one that motivated this requirement assumed a butler that mediated a lot of database access.  Since then we've move toward approaches that us parquet instead, and it's not clear that butler should be playing a role in those workflows.
```

Comment:
I lean towards dropping this requirement.

Reply:
At least drop the priority. This was driven by SQuaRE so I wonder if @SimonKrughoff feels it would still be useful.

Reply:
I'd drop this to a P2 - it's not necessary it's a nice to have